File: 0268-missing-number/0268-missing-number.cpp

class Solution {
public:
    int missingNumber(vector<int>& nums) 
    {
        sort(nums.begin(), nums.end()); 
        
        for(int i=0;i<=nums.size()-1;i++)
        {
            if(binary_search(nums.begin(),nums.end(),i)==0)
                return i;
        }
        return nums.size();
        
    }
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        if (len(nums)==1 and nums[0]==1):
            return 0;
        
        for i in range(0,max(nums)+1):
            if i not in nums:
                return (i)
        return len(nums)
        